### DISCUSSION TITLE
How to put navbar back

### DISCUSSION THREAD

**Discussion by timmeade** [fuma-nama/fumadocs#489] [General]
I just discovered this package yesterday and have been playing with it this morning.  There is a lot of promise here.  

I have read the issue about ui redesign and the removal of the navbar.  I get it.  Some sites that is absolutely the way to go.  But some a navbar is 100% required.   The one I am implementing now has a navbar with the search centered.  

How do I accomplish that now?   I must match the design and if i cannot do that i cannot use this.   Am i missing something?

**Comment by fuma-nama** [comment]
Honestly, I have not planned for supporting it. The navbar is currently a mobile-only element and I avoided adding more customisable options, it takes way more efforts to keep the customised UI and its different variants look correct while making changes to the UI.

A CSS selector can hide the search toggle from sidebar, and you can pass a custom navbar to the nav option of docs layout. However, you might need to copy the code of some components to make it work. (My recommended approach since I'm improving the UI regularly)
